From a152a6e09d5e8d3e6ffbb6e01a378d38eaaa5807 Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Thu, 3 Jan 2019 03:16:11 +0100
Subject: [PATCH] gluon-config-mode-geo-location: fix source/i18n mismatch

The text was changed in the i18n files, but the corresponding change in the
Lua sources seemingly got lost during a rebase.

Closes #1611

(cherry picked from commit 2aa324ecf7f38ea2a59aee0eaf745c54c50858bb)
---
 .../luasrc/lib/gluon/config-mode/wizard/0400-geo-location.lua   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/gluon-config-mode-geo-location/luasrc/lib/gluon/config-mode/wizard/0400-geo-location.lua b/package/gluon-config-mode-geo-location/luasrc/lib/gluon/config-mode/wizard/0400-geo-location.lua
index 6a211b64..928b4c5a 100644
--- a/package/gluon-config-mode-geo-location/luasrc/lib/gluon/config-mode/wizard/0400-geo-location.lua
+++ b/package/gluon-config-mode-geo-location/luasrc/lib/gluon/config-mode/wizard/0400-geo-location.lua
@@ -15,7 +15,7 @@ return function(form, uci)
 	if not text then
 		text = pkg_i18n.translate(
 			'If you want the location of your node to ' ..
-			'be displayed on the map, you can enter its coordinates here.'
+			'be displayed on public maps, you can enter its coordinates here.'
 		)
 		if osm then
 			text = text .. ' ' .. osm.help(i18n)
-- 
2.17.1

